apps/trading/e2e/tests/deal_ticket/test_isolated_cross_margin.py

```diff
@@ -51,3 +57,47 @@ def test_switch_cross_isolated_margin(
     next_epoch(vega=vega)
     expect(page.locator(margin_row).nth(1)).to_have_text(
         "22,109.99996Cross1.0x")
+
+
+@pytest.mark.usefixtures("auth", "risk_accepted")
+def test_check_cross_isolated_margin_info(
+        continuous_market, vega: VegaServiceNull, page: Page):
+    create_position(vega, continuous_market)
+    page.goto(f"/#/markets/{continuous_market}")
+    expect(page.get_by_test_id(current_margin)).to_have_text(
+        "Current margin874.21992 tDAI")
+    expect(page.get_by_test_id(available_collaterals)
+           ).to_have_text("Available collateral998,084.95183 tDAI")
+    expect(page.get_by_test_id(additional_margin_requiredd)
+           ).to_have_text("Additional margin required0.00 tDAI")
+    expect(page.get_by_test_id(liquidation_estimate)
+           ).to_have_text("Liquidation estimate- BTC")
+    page.get_by_role("button", name="Isolated 10x").click()
```

Comment:
If you wanted to make things a bit nicer, you could add a testID for the button so you dont need to get_by_role.

Reply:
I added a new testid but it also has its advantage because it checks whether the text on the button is correct